crimsobot/utils/image.py

```diff
@@ -115,6 +116,12 @@ async def fetch_image(ctx: Context, arg: Optional[str]) -> Image.Image:
     session = aiohttp.ClientSession()
 
     async def open_img_from_url(url: str) -> Image.Image:
+        if 'tenor.com/view' in url:
+            async with session.get(url, allow_redirects=False) as response:
```

Comment:
This is WET. I shouldn't need two `session.get()`s to get the job done here, but I can't quite figure out how to do this with a single `session.get()`. I blame my unfamiliarity with aiohttp.

Reply:
nah it's fine. you have to make two requests anyway, no way around it